<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>441</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>521</width>
    <height>441</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>521</width>
    <height>441</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Персистентный стек</string>
  </property>
  <widget class="QPushButton" name="btn_open_add">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_delete">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_delete_all">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>400</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Удалить всё</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Выбор версии:</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>381</width>
     <height>361</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="movement">
    <enum>QListView::Snap</enum>
   </property>
   <property name="flow">
    <enum>QListView::TopToBottom</enum>
   </property>
   <property name="resizeMode">
    <enum>QListView::Fixed</enum>
   </property>
   <property name="layoutMode">
    <enum>QListView::SinglePass</enum>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="viewMode">
    <enum>QListView::ListMode</enum>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
